httpbin/handlers.go

```diff
@@ -23,7 +23,8 @@ func notImplementedHandler(w http.ResponseWriter, r *http.Request) {
 // Index renders an HTML index page
 func (h *HTTPBin) Index(w http.ResponseWriter, r *http.Request) {
 	if r.URL.Path != "/" {
-		http.Error(w, "Not Found", http.StatusNotFound)
+		msg := fmt.Sprintf("Not Found (go-httpbin does not have the API: %s)", r.URL.Path)
```

Comment:
How about 

> Not Found (go-httpbin does not handle the path %s)

as a slight tweak to the message? I just don’t think of these paths/routes as “APIs”.

Reply:
I agree with you, thanks, have adjusted the code.